packages/blocks/src/api/registration.js

```diff
@@ -228,6 +228,18 @@ export function registerBlockType( name, settings ) {
 	return settings;
 }
 
+/**
+ * Registers a new block collection to group blocks in the same namespace in the inserter.
+ *
+ * @param {string} namespace The namespace to group blocks by in the inserter; corresponds to the block namespace
+ * @param {Object} tite      The title to show in the inserter
+ * @param {Object} icon      The icon to show in the inserter
+ *
+ */
+export function registerBlockCollection( namespace, title, icon ) {
```

Comment:
All the existing public API methods which register something use settings object as 2nd param, it would be great to align, see:

https://github.com/WordPress/gutenberg/blob/f8b62bd29fc83972256d7b2315dee2341af06cc1/packages/rich-text/src/register-format-type.js#L36
https://github.com/WordPress/gutenberg/blob/f8b62bd29fc83972256d7b2315dee2341af06cc1/packages/plugins/src/api/index.js#L109
https://github.com/WordPress/gutenberg/blob/f8b62bd29fc83972256d7b2315dee2341af06cc1/packages/blocks/src/api/registration.js#L119

All of them offer `unregister*` function as well which returns the registered object for more control for 3rd party plugings.

Reply:
Done!